cob_bringup/components/ros_control_base.launch

```diff
@@ -25,10 +26,12 @@
 		<arg name="reset_errors_before_recover" value="$(arg reset_errors_before_recover)"/>
 	</include>
 
-	<include unless="$(arg sim)" file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
-		<arg name="timeout" default="2.0"/>
-		<arg name="threshold" default="0.174533"/>
-	</include>
+	<group if="$(arg enable_stuck_detector)">
+		<include unless="$(arg sim)" file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
+			<arg name="timeout" default="2.0"/>
```

Comment:
@ipa-fez: could you successfully use the stuck detector with adapted parameters? if yes, we should remove the parameters from this launch file and move them to a robot specific yaml file in cob_hardware_config and upload that here.